### DISCUSSION TITLE
Brain state: Deep Sleeping Errors

### DISCUSSION THREAD

**Discussion by Roozeboom007** [NightscoutFoundation/xDrip#2184] [General]
I'm using one of the latest versions of xDrip, but it loses connection every now and then. Restart the phone, then i am connection again.
De error is "Brain state: Deep Sleeping Errors"
I have a OnePlus 7T with Android version 11, Sensor transmitter is 8S....
What's the problem here?

The settings used are according to this:
https://navid200.github.io/xDrip/docs/Dexcom_page.html

![Screenshot_20220621-070933__01](https://user-images.githubusercontent.com/58114624/175537452-32b2a5ae-5bd9-4110-b91c-c60e495047f1.jpg)
![Screenshot_20220621-070923__01](https://user-images.githubusercontent.com/58114624/175537473-2382dc19-bcd8-4daa-8d08-795a87c6ab60.jpg)
![Screenshot_20220621-070914](https://user-images.githubusercontent.com/58114624/175537483-7fe5390c-a239-4dc7-842f-b1c2eb13b535.jpg)
![Screenshot_20220621-070902](https://user-images.githubusercontent.com/58114624/175537496-3224c98a-c5f0-4bf0-bf26-5d6fcb6f6a73.jpg)
![Screenshot_20220622-093910__01](https://user-images.githubusercontent.com/58114624/175537506-5331e6fd-89d9-42df-b173-f4ce4c259395.jpg)

**Comment by Navid200** [comment]
Have you disabled xDrip and Bluetooth battery optimization?

**Comment by Roozeboom007** [comment]
I have both set to don't optimize
